readthedocs/projects/views/base.py

```diff
@@ -30,7 +30,7 @@ def get_context_data(self, **kwargs):
 
         # Show for the first few builds, return last build state
         if project.builds.count() <= 5:
-            onboard["build"] = project.get_latest_build(finished=False)
+            onboard["build"] = project.latest_internal_build
```

Comment:
It seems there is some change in behavior here. We were returning unfinished builds before, and now we are returning anyone. Is that expected?

Reply:
When finished was false it was returning the latest internal build, so no change in behavior here.

Reply:
Ah, you are right. I checked the code for `finished=True` and there is no usage of it. It seems the method was prepared to support this behavior but it wasn't used 👍🏼